src/Tizen.NUI/src/public/Theme/ThemeManager.cs

```diff
@@ -25,6 +25,7 @@ namespace Tizen.NUI
     internal interface IThemeCreator
     {
         Theme Create();
+        Theme Create(IEnumerable<KeyValuePair<string, object>> changedResources);
```

Comment:
if you don't mind, could you remove the warning below?

src/Tizen.NUI/src/public/Theme/ThemeManager.cs#L45
Unused field 'nuiThemeProjects'.

Reply:
warning message `Unused field 'nuiThemeProjects'.` is not related this commit. 
To keep atomic commits, i think we can handle this warning in another patch.